src/components/approval-type/ApprovalType.vue

```diff
@@ -64,53 +105,86 @@ export default class ApprovalType extends Vue {
   /** Show only the court order option; remove via registrar option. */
   @Prop({ default: false }) readonly isCourtOrderOnly!: boolean
 
+  /** Draft notice date. */
+  @Prop({ default: '' }) readonly notice!: string
+
+  /** Draft application date. */
+  @Prop({ default: '' }) readonly application!: string
+
   // Local properties
   private courtOrderNumber = ''
   private courtOrderNumRules = []
   private valid = false
   private approvalTypeSelected = ''
+  private datePickerRules = []
+  private noticeDateText = ''
+  private applicationDateText = ''
 
   /** Called when component is mounted. */
   mounted (): void {
+    this.setDateRules()
+    this.setCourtNumberRules()
     // Copy props to mutable properties
     if (this.draftApprovedByRegistrar) {
       this.courtOrderNumber = ''
       this.approvalTypeSelected = this.ApprovalTypes.VIA_REGISTRAR
-    }
-    if (this.draftCourtOrderNumber) {
+    } else if (this.draftCourtOrderNumber) {
       this.courtOrderNumber = this.draftCourtOrderNumber
       this.approvalTypeSelected = this.ApprovalTypes.VIA_COURT_ORDER
+    } else {
+      // Default state (no button selected)
+      this.radioButtonChanged('')
     }
   }
 
   /** Triggers the form validation. */
   public validate (): boolean {
-    let status = this.$refs.courtNumRef.validate()
-    this.$emit('emitValid', status)
-    return status
+    if (this.approvalTypeSelected === this.ApprovalTypes.VIA_COURT_ORDER) {
+      let status = this.$refs.courtNumRef.validate()
+      this.$emit('emitValid', status)
+      return status
+    } else if (this.approvalTypeSelected === this.ApprovalTypes.VIA_REGISTRAR) {
+      // Emit true (valid) if both dates were selected. Emit false (invalid) if at least one was empty.
+      let status = (!!this.noticeDateText && !!this.applicationDateText)
+      this.$emit('emitValid', status)
```

Comment:
I know you're just following what was already here, but we typically use a `@Emit`-decorated method to emit events. (Consistency = easier to understand.)

Also, events probably shouldn't be named "emitSomething". In the case of validity, you can just emit a "valid" event.

On the other hand, if you're emitted an update to a prop (eg, notice date), then you can do:
```
@Emit('update:noticeDate')
private emitNoticeDate (noticeDate: string): string {
   return noticeDate
}
```
or
```
@Emit('update:noticeDate')
private emitNoticeDate (noticeDate: string): void {}
```